addons/website_sale/views/templates.xml

```diff
@@ -813,6 +814,39 @@
                         </a>
                     </li>
                 </ul>
+                <ul t-else="" class="d-flex gap-2 list-unstyled">
+                    <t t-foreach="entries" t-as="c" t-if="c.image_128" t-set="atLeastOneImage" t-value="True"/>
+                    <t t-if="category.parent_id" t-set="backUrl" t-value="keep('/shop/category/' + slug(category.parent_id), category=0)"/>
+                    <t t-else="" t-set="backUrl" t-value="'/shop'"/>
```

Comment:
Is there a reason why these computations are duplicated in both "blocks" ?
Could me moved just after `o_wsale_filmstip_wrapper` instead?